big_tests/dynamic_domains.spec

```diff
@@ -31,9 +31,9 @@
 {suites, "tests", inbox_extensions_SUITE}.
 
 {suites, "tests", mam_SUITE}.
-{skip_cases, "tests", mam_SUITE,
- [muc_service_discovery, mam_service_discovery],
- "at the moment mod_mam config options don't support dynamic domains"}.
+%{skip_cases, "tests", mam_SUITE,
```

Comment:
Could you remove these lines completely?

Reply:
sure, will do a separate PR to master, once big disco PR merged.